src/dcommands/timer.js

```diff
@@ -31,10 +31,8 @@ class TimerCommand extends BaseCommand {
     }
 
     async event(args) {
-        let duration = moment.duration(moment() - moment(args.starttime));
-        duration.subtract(duration * 2);
         bu.send(args.channel, {
-            content: `:alarm_clock: *Bzzt!* <@${args.user}>, the timer you set ${duration.humanize(true)} has gone off! *Bzzt!* :alarm_clock:`,
+            content: `:alarm_clock: *Bzzt!* <@${args.user}>, the timer you set <t:${args.starttime}:R> has gone off! *Bzzt!* :alarm_clock:`,
```

Comment:
```suggestion
            content: `:alarm_clock: *Bzzt!* <@${args.user}>, the timer you set <t:${moment(args.starttime).unix()}:R> has gone off! *Bzzt!* :alarm_clock:`,
```
The discord timestamp can also be added to the message when a timer has been created o/ Pretty much identical to the `b!remind` implementation